### ISSUE TITLE
Not able to parse @BodyParam

### ISSUE THREAD

**Issue by AlfieriChou** [epiphone/routing-controllers-openapi#14]
```javascript
  async create(
    @BodyParam('title', { required: true }) title: string,
    @BodyParam('text') text: string
  ) {
    const demoRepository = getManager().getRepository(Demo)
    const newDemo = await demoRepository.create({
      title: title,
      text: text || ''
    })
    const result = await demoRepository.save(newDemo)
    return result
  }
```
but this api doc is No parameters

**Comment by epiphone** [comment]
Hi! Good point, `@BodyParam` is not currently supported. I'd be happy to merge in a PR if anyone feels like implementing this!

**Comment by epiphone** [comment]
Support for `BodyParam` released in `1.5.0`.
